apps/condo/domains/contact/access/Contact.js

```diff
@@ -7,16 +7,28 @@ const get = require('lodash/get')
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById } = require('@open-condo/keystone/schema')
 
+const {
+    b2bAppServiceUserCanReadObjects,
+    b2bAppServiceUserCanManageObjects,
+} = require('@condo/domains/miniapp/utils/b2bAppServiceUserAccess')
 const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
 const { queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
 const { queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+
+async function canReadContacts (args) {
+    const { authentication: { item: user } } = args
 
-async function canReadContacts ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     
     if (user.isAdmin) return {}
 
+    if (user.type === SERVICE) {
+        return await b2bAppServiceUserCanReadObjects(args)
```

Comment:
How about something like this to stay consistent with naming?
```suggestion
        return await canReadObjectsAsB2BAppServiceUser(args)
```